permissions/plugin-gitea-checks.yml

```diff
@@ -2,8 +2,10 @@
 name: "gitea-checks"
 github: &GH "jenkinsci/gitea-checks-plugin"
 paths:
-- "com/github/flozzone/gitea-checks"
+- "org/jenkins-ci/plugins/gitea-checks"
```

Comment:
This is the legacy ID, if you’re changing it you’re supposed to use io.jenkins.plugins